changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2nd November 2022
+
+* Extended [Outlet item](../operations/outletitems.md) with `ExternalIdentifier`.
+* Extended [Outlet bill](../operations/outletbills.md) with `ExternalIdentifier` for items.
```

Comment:
The reference here should be [Outlet bill parameters](../operations/outletbills.md#outlet-bill-parameters).